Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -1012,6 +1012,16 @@ - (void)notifyAPNSTokenIsSet:(NSNotification *)notification {
     FIRInstanceID_WEAKIFY(self);
     [self asyncLoadKeyPairWithHandler:^(FIRInstanceIDKeyPair *keyPair, NSError *error) {
       FIRInstanceID_STRONGIFY(self);
+      if (self == nil) {
+        FIRInstanceIDLoggerDebug(kFIRInstanceIDMessageCodeInstanceID017, 
```

Comment:
Should this be warning/error level at least? ( same as below)

Reply:
Done.